lib/CMakeLists.txt

```diff
@@ -39,7 +39,8 @@ include(${CMAKE_CURRENT_BINARY_DIR}/conan.cmake)
 list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_BINARY_DIR})
 list(APPEND CMAKE_PREFIX_PATH ${CMAKE_CURRENT_BINARY_DIR})
 
-if(SKBUILD)
+option(CONAN_TBB "Use TBB from conan" OFF)
+if(SKBUILD OR CONAN_TBB)
   # Conda builds install tbb via conda, not conan
   set(CONAN_ONETBB onetbb/2021.3.0)
```

Comment:
Didn't notice before, but this is a different version from conda (2020.2). Should we use the same everywhere?

Reply:
2020.2 is because of conda-forge and Mantid. Is there any reason sticking to an older one otherwise?

Reply:
Only for consistency. Are we testing on CI with both versions? If not, the conda package might have a bug that we don't detect because the tests run with the pip package, or vice versa.

Reply:
I believe `conda build` is running the tests?

Reply:
For the package, yes. Do you think the risk is low enough to rely on package builds and no CI?

Reply:
We are not using/changing much for TBB, so I'd say yes?